diesel/src/type_impls/primitives.rs

```diff
@@ -216,7 +192,7 @@ where
     DB: Backend,
     T::Owned: FromSql<ST, DB>,
 {
-    fn from_sql(bytes: Option<&DB::RawValue>) -> deserialize::Result<Self> {
+    fn from_sql<'b>(bytes: Option<<<DB as Backend>::RawValue as FamilyLt<'b>>::Out>) -> deserialize::Result<Self> {
```

Comment:
```suggestion
    fn from_sql<'b>(bytes: Option<RawValue<'b, DB>>) -> deserialize::Result<Self> {
```

Reply:
This doesn't seem to be a valid change because the expected type is:
fn(std::option::Option<<::RawValue as backend::FamilyLt<'a>>::Out>)